webknossos/webknossos/cli/upload.py

```diff
@@ -42,7 +42,14 @@ def main(
         str | None,
         typer.Option(
             help="Alternative name to rename your dataset on upload to WEBKNOSSOS. "
-            "(if not provided, current name of dataset is used)",
+            "If not provided, current name of dataset is used.",
+        ),
+    ] = None,
+    folder: Annotated[
+        str | None,
+        typer.Option(
+            help="Folder where the dataset is uploaded to WEBKNOSSOS. Specify the folder name as a string, separated by `/`. "
```

Comment:
```suggestion
            help="WEBKNOSSOS dataset folder in which the dataset should be placed. Specify the folder path as a string, separated by `/`. Example: Datasets/mySubfolder"
```
(or should there be a leading slash?)